src/cljs/rems/catalogue.cljs

```diff
@@ -18,9 +18,8 @@
  ::enter-page
  (fn [{:keys [db]} _]
    (if (roles/is-logged-in? (get-in db [:identity :roles]))
```

Comment:
I think you want to delete this `if` and the `unauthorized!` below. The dispatch is needed, it causes the call to `/api/catalogue`